src/Symfony/Component/Translation/Extractor/Visitor/AbstractVisitor.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Extractor\Visitor;
+
+use PhpParser\Node;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * @author Mathieu Santostefano <msantostefano@protonmail.com>
+ */
+abstract class AbstractVisitor
+{
+    protected MessageCatalogue $catalogue;
+    protected \SplFileInfo $file;
+    protected string $messagePrefix;
```

Comment:
should those be protected ? Do we actually want child classes to access them directly ?